app/models/grda_warehouse/hud/project.rb

```diff
@@ -292,7 +292,8 @@ def serves_children?
     #   within the context of reporting confidential_scope_limiter is almost always non_confidential
     #   within the client dashboard context, confidential_scope_limiter is :all, which includes confidential projects
     #   names of confidential projects are obfuscated unless the user can_view_confidential_project_names
-    # @param permission [Symbol] a permission to determine the scope for which the projects are viewable
+    # @param permission [Symbol] a permission to determine the scope for which the projects are viewable.
+    #   Caution, this permission is NOT checked if the user is using legacy permissions (not on ACLs).
```

Comment:
todo @martha - add comment - need to add the check for permissions somewhere else, because in role based, it's not related to the specific entity.